packages/rum/test/specs/apm-base.spec.js

```diff
@@ -67,6 +67,31 @@ describe('ApmBase', function() {
     })
   })
 
+  it('should be noop when agent is not active', done => {
```

Comment:
What will happen if users call `startTransaction` on an inactive agent. We should have a test for that as well.